client/shared/helpers/listen.coffee

```diff
@@ -7,9 +7,9 @@ EventBus = require 'shared/services/event_bus'
 module.exports =
   listenForMentions: ($scope, model) ->
     $scope.unmentionableIds = [model.authorId, Session.user().id]
```

Comment:
Hm, I wonder if we could get rid of this client-side check now; we should be filtering out these users on the server I reckon. Then this turns into

```
$scope.fetchByNameFragment = (q) ->
  Records.users.fetchMentionable(q).then (response) ->
    $scope.mentionables = Records.users.find(_.pluck(response.users, 'id'))
```